cmd/beaker/session.go

```diff
@@ -233,55 +291,126 @@ func newSessionUpdateCommand() *cobra.Command {
 	return cmd
 }
 
-func awaitSessionSchedule(session string) (*api.Session, error) {
-	if !quiet {
-		fmt.Printf("Waiting for session to be scheduled")
+func awaitSessionSchedule(session api.Session) (*api.Session, error) {
+	s := beaker.Session(session.ID)
+	cl := beaker.Cluster("ai2/" + session.Cluster)
```

Comment:
So, after the API change this will just be:

```
cl := beaker.Cluster(session.Cluster)
```

...right?

Reply:
Right. I put this in for testing and forgot to revert it on my second commit. Should be...

```suggestion
	cl := beaker.Cluster(session.Cluster)
```